internal/controller/sonataflowplatform_controller_test.go

```diff
@@ -919,7 +919,6 @@ func TestSonataFlowPlatformController(t *testing.T) {
 		validateTrigger(t, cl, "data-index-process-definition-", ksp.Namespace, ksp, trigger)
 		validateTrigger(t, cl, "data-index-process-error-", ksp.Namespace, ksp, trigger)
 		validateTrigger(t, cl, "data-index-process-node-", ksp.Namespace, ksp, trigger)
-		validateTrigger(t, cl, "data-index-process-sla-", ksp.Namespace, ksp, trigger)
```

Comment:
@ricardozanini Btw, the "data-index-process-sla-" was basically removed because the workflows don't produce this event, so we have basically one lees k8s object to create/maintain, etc.